gradle.properties

```diff
@@ -15,21 +15,19 @@
 #
 # project metadata used for publications
 group=io.servicetalk
-version=0.42.0
+version=0.42.11-SNAPSHOT
```

Comment:
This will be updated to `0.42.11` once it is released. The `servicetalk-dependencies` BOM is not available in earlier releases.